spanner_orm/field.py

```diff
@@ -148,6 +192,14 @@ def validate_type(value) -> None:
     if not isinstance(value, str):
       raise error.ValidationError('{} is not of type str'.format(value))
 
+  @staticmethod
+  def matches(type: str) -> bool:
+    return type.startswith('STRING(') and type.endswith(')')
```

Comment:
(nit) I'd use a regular expression here to avoid matching something like `STRING(MAX) other_column STRING(MAX)` if somebody forgets a comma between two column definitions or something.